cmd/kube-apiserver/app/options/options.go

```diff
@@ -130,11 +131,11 @@ func gvToMap(gvList string, dest map[string]string) {
 		// "group=group/version". The latter is used when types
 		// move between groups.
 		if !strings.Contains(gv, "=") {
-			dest[apiutil.GetGroup(gv)] = gv
+			dest[apiutil.GetGroup(gv)] = unversioned.GroupVersion{Group: apiutil.GetGroup(gv), Version: apiutil.GetVersion(gv)}
```

Comment:
> Is there not a function that constructs a GroupVersion from a string?

Not with the same (broken) rules as `apiutil` that this file used previously.  How compatible would you like to be?  If I'm allowed to be incompatible, I can do a better job here.